lib/evmone/instructions_calls.cpp

```diff
@@ -7,7 +7,7 @@
 namespace evmone::instr::core
 {
 template <Opcode Op>
-evmc_status_code call_impl(StackTop stack, ExecutionState& state) noexcept
+Result call_impl(StackTop stack, int64_t gas_left, ExecutionState& state) noexcept
```

Comment:
Nit: it's called `gas` in `invoke` wrappers, should it be also `gas` here?
```suggestion
Result call_impl(StackTop stack, int64_t gas, ExecutionState& state) noexcept
```

Reply:
I'm switching from `gas_left` to `gas` because seems good enough.

Reply:
I will leave `gas_left` in instructions this time because it comes from `state.gas_left`. This slightly limits the number of change.s